src/modules/Transcript/index.tsx

```diff
@@ -17,40 +17,48 @@ import './styles.css';
 type Event = TranscriptQuery['events'][0];
 type Paragraph = TranscriptQuery['events'][0]['transcripts'][0]['sections'][0]['speakerTurns'][0]['paragraphs'][0];
 interface TranscriptUIProps {
+    headerExpanded: boolean;
+    toggleHeader: () => void;
     event?: Event;
     onBack?: MouseEventHandler;
     paragraphs: Paragraph[];
 }
 
 export const TranscriptUI = (props: TranscriptUIProps): ReactElement => {
-    const { event, onBack, paragraphs } = props;
+    const { event, onBack, paragraphs, toggleHeader, headerExpanded } = props;
     const primaryQuote = getPrimaryQuote(event?.primaryCompany);
     const eventDate = DateTime.fromISO(event?.eventDate);
-    const [showGear, setGear] = useState(false);
     return (
         <div className="h-full flex flex-col transcript">
-            <div className="relative p-3 shadow-header rounded-b-lg transcript__header">
+            <div
+                className={
+                    headerExpanded
+                        ? 'max-h-80 relative p-3 shadow-header rounded-b-lg transition-all transcript__header'
```

Comment:
Generally I try to keep all the common ones together and only do the conditional for the ones that change to it's more clear, so something like:
```
classNames(
   'relative p-3 shadow-header rounded-b-lg transition-all transcript__header',
   headerExpanded ? 'max-h-80' : 'max-h-28'
)
```